app/scripts/controllers/app-state-controller.ts

```diff
@@ -196,79 +202,239 @@ const getDefaultAppStateControllerState = (
   currentExtensionPopupId: 0,
 });
 
-export class AppStateController extends EventEmitter {
-  private readonly extension: AppStateControllerOptions['extension'];
+const controllerMetadata = {
+  timeoutMinutes: {
+    persist: true,
+    anonymous: true,
+  },
+  connectedStatusPopoverHasBeenShown: {
+    persist: true,
+    anonymous: true,
+  },
+  defaultHomeActiveTabName: {
+    persist: true,
+    anonymous: true,
+  },
+  browserEnvironment: {
+    persist: true,
+    anonymous: true,
+  },
+  popupGasPollTokens: {
+    persist: true,
+    anonymous: true,
+  },
+  notificationGasPollTokens: {
+    persist: true,
+    anonymous: true,
+  },
+  fullScreenGasPollTokens: {
+    persist: true,
+    anonymous: true,
+  },
+  recoveryPhraseReminderHasBeenShown: {
+    persist: true,
+    anonymous: true,
+  },
+  recoveryPhraseReminderLastShown: {
+    persist: true,
+    anonymous: true,
+  },
+  outdatedBrowserWarningLastShown: {
+    persist: true,
+    anonymous: true,
+  },
+  nftsDetectionNoticeDismissed: {
+    persist: true,
+    anonymous: true,
+  },
+  showTestnetMessageInDropdown: {
+    persist: true,
+    anonymous: true,
+  },
+  showBetaHeader: {
+    persist: true,
+    anonymous: true,
+  },
+  showPermissionsTour: {
+    persist: true,
+    anonymous: true,
+  },
+  showNetworkBanner: {
+    persist: true,
+    anonymous: true,
+  },
+  showAccountBanner: {
+    persist: true,
+    anonymous: true,
+  },
+  trezorModel: {
+    persist: true,
+    anonymous: true,
+  },
+  currentPopupId: {
+    persist: false,
+    anonymous: true,
+  },
+  onboardingDate: {
+    persist: false,
+    anonymous: true,
+  },
+  lastViewedUserSurvey: {
+    persist: true,
+    anonymous: true,
+  },
+  newPrivacyPolicyToastClickedOrClosed: {
+    persist: true,
+    anonymous: true,
+  },
+  newPrivacyPolicyToastShownDate: {
+    persist: true,
+    anonymous: true,
+  },
+  hadAdvancedGasFeesSetPriorToMigration92_3: {
+    persist: true,
+    anonymous: true,
+  },
+  qrHardware: {
+    persist: false,
+    anonymous: true,
+  },
+  nftsDropdownState: {
+    persist: false,
+    anonymous: true,
+  },
+  usedNetworks: {
+    persist: false,
```

Comment:
Hmm. This one _was_ getting reset each time the extension started, but I think that was a bug. This tracks when a user used a network for the first time. We're causing the "first-time modal" to re-appear every session because this is getting wiped.

We should confirm with the assets team what was expected here

Reply:
@MetaMask/metamask-assets can we confirm this behavior ?

Reply:
Yes, @Gudahtt is correct. This might be a bug where the educational modal appears every time the extension is reloaded and the network is added again. 
However, this educational modal has been removed in this [PR](https://github.com/MetaMask/metamask-extension/pull/28765), so I believe the usedNetworks is no longer needed.

Reply:
@Gudahtt I just made a PR to completely remove the property from the state https://github.com/MetaMask/metamask-extension/pull/28813